manifests/j/JetBrains/Kotlin/Compiler/1.3.31/JetBrains.Kotlin.Compiler.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with Komac v1.9.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JetBrains.Kotlin.Compiler
+PackageVersion: 1.3.31
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: kotlinc\bin\kotlinc.bat
+  PortableCommandAlias: kotlinc
+- RelativeFilePath: kotlinc\bin\kotlinc-js.bat
+  PortableCommandAlias: kotlin-js
+- RelativeFilePath: kotlinc\bin\kotlin.bat
+  PortableCommandAlias: kotlin
+- RelativeFilePath: kotlinc\bin\kapt.bat
+  PortableCommandAlias: kapt
+- RelativeFilePath: kotlinc\bin\kotlin-dce-js.bat
+  PortableCommandAlias: kotlin-dce-js
+- RelativeFilePath: kotlinc\bin\kotlinc-jvm.bat
+  PortableCommandAlias: kotlin-jvm

```

Comment:
```suggestion
- RelativeFilePath: kotlinc\bin\kotlinc
  PortableCommandAlias: kotlinc
- RelativeFilePath: kotlinc\bin\kotlinc-js
  PortableCommandAlias: kotlin-js
- RelativeFilePath: kotlinc\bin\kotlin
  PortableCommandAlias: kotlin
- RelativeFilePath: kotlinc\bin\kotlin-dce-js
  PortableCommandAlias: kotlin-dce-js
- RelativeFilePath: kotlinc\bin\kotlinc-jvm
  PortableCommandAlias: kotlin-jvm
```
These files don't have the `.bat` extension in this version. I do not know if this will work. 
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/cb9d69c4-cd2d-4991-84f0-a0c88e83f3bf)